2.segment_images/scripts/2.segment_whole_organoids.py

```diff
@@ -120,9 +177,9 @@
 
 cyto = np.max(
     [
-        # cyto1,
-        cyto2,
-        cyto3,
+        # cyto1, # ER
+        cyto2,  # AGP
+        # cyto3, # Mito
```

Comment:
Consider renaming to `channel_agp` or something similar to help make this more specific. Also - why only this channel?